edalize/symbiflow.py

```diff
@@ -52,17 +54,17 @@ def get_doc(cls, api_ver):
                     {
                         "name": "pnr",
                         "type": "String",
-                        "desc": 'Place and Route tool. Currently only "vpr" is supported',
+                        "desc": 'Place and Route tool. Currently only "vpr"/"vtr" and "nextpnr" are supported',
                     },
                     {
                         "name": "vpr_options",
                         "type": "String",
                         "desc": "Additional vpr tool options. If not used, default options for the tool will be used",
                     },
                     {
-                        "name" : "environment_script",
-                        "type" : "String",
-                        "desc" : "Optional bash script that will be sourced before each build step."
+                        "name": "environment_script",
+                        "type": "String",
+                        "desc": "Optional bash script that will be sourced before each build step."
                     },
```

Comment:
I would like to remove the enviromnent_script options and use the EDALIZE_LAUNCHER mechanism that I added for some other backends a while ago which is more generic

Reply:
I've done some clean up and removed this option along with `SOURCE_OPTS` from makefiles.